src/frequenz/sdk/data_ingestion/resampling/component_metric_resampler.py

```diff
@@ -0,0 +1,100 @@
+"""
+ComponentMetricResampler class for resampling individual metrics.
+
+Copyright
+Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+import logging
+from collections import deque
+from datetime import datetime, timedelta
+from typing import Callable, Deque, Optional, Sequence
+
+import pytz
+
+from frequenz.sdk.data_pipeline import Sample
```

Comment:
Please use relative imports for local modules (`from ..data_pipeline import Sample`).

Reply:
Done